### PR TITLE
Fix `#version` method docs and some typos [ci-skip]

### PR DESCRIPTION
**Pull request by ghousemohamed**
### Summary

Made the doc for the `#version` method more consistent and fixed a few typos.

### CODE REVIEW

File: actionpack/lib/action_pack/version.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 require_relative "gem_version"
 
 module ActionPack
-  # Returns the version of the currently loaded ActionPack as a <tt>Gem::Version</tt>
+  # Returns the version of the currently loaded Action Pack as a <tt>Gem::Version</tt>.
```
Comment (rafaelfranca) [inline actionpack/lib/action_pack/version.rb:6]:
```suggestion
  # Returns the currently loaded version of Action Pack as a <tt>Gem::Version</tt>.
```

File: actionview/lib/action_view/version.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 require_relative "gem_version"
 
 module ActionView
-  # Returns the version of the currently loaded ActionView as a <tt>Gem::Version</tt>
+  # Returns the version of the currently loaded Action View as a <tt>Gem::Version</tt>.
```
Comment (rafaelfranca) [inline actionview/lib/action_view/version.rb:6]:
```suggestion
  # Returns the currently loaded version of Action View as a <tt>Gem::Version</tt>.
```

File: activerecord/lib/active_record/version.rb [hunk 3]
```diff
@@ -3,7 +3,7 @@
 require_relative "gem_version"
 
 module ActiveRecord
-  # Returns the version of the currently loaded ActiveRecord as a <tt>Gem::Version</tt>
+  # Returns the version of the currently loaded Active Record as a <tt>Gem::Version</tt>.
```
Comment (rafaelfranca) [inline activerecord/lib/active_record/version.rb:6]:
```suggestion
  # Returns the currently loaded version of Active Record as a <tt>Gem::Version</tt>.
```

File: activestorage/lib/active_storage/version.rb [hunk 4]
```diff
@@ -3,7 +3,7 @@
 require_relative "gem_version"
 
 module ActiveStorage
-  # Returns the version of the currently loaded ActiveStorage as a <tt>Gem::Version</tt>
+  # Returns the version of the currently loaded Active Storage as a <tt>Gem::Version</tt>.
```
Comment (rafaelfranca) [inline activestorage/lib/active_storage/version.rb:6]:
```suggestion
  # Returns the currently loaded version of Active Storage as a <tt>Gem::Version</tt>.
```

File: activesupport/lib/active_support/version.rb [hunk 5]
```diff
@@ -3,7 +3,7 @@
 require_relative "gem_version"
 
 module ActiveSupport
-  # Returns the version of the currently loaded ActiveSupport as a <tt>Gem::Version</tt>
+  # Returns the version of the currently loaded Active Support as a <tt>Gem::Version</tt>.
```
Comment (rafaelfranca) [inline activesupport/lib/active_support/version.rb:6]:
```suggestion
  # Returns the currently loaded version of Active Support as a <tt>Gem::Version</tt>.
```

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

**Comment by ghousemohamed** [comment]
@rafaelfranca I've made the changes as you have suggested.
I have a query related to the `#version` and `#gem_version` methods. Both the methods return the same value. Shouldn't there be a distinction between the two methods? For example, in `railties`, I can see that `#version` returns `VERSION::STRING` whereas `#gem_version` returns Gem::Version.new VERSION::STRING`. Here there is a clear distinction between the two methods. Should this be the case for all other gems?

**Comment by rafaelfranca** [comment]
No. [The Rails `version` returns a string because of backwards compatibility. `version` is also in the other components there also for backwards compatibility since it doesn't add much value deprecate i](https://github.com/rails/rails/pull/14101)t. We could say `version` should return strings, but that will break someones code and it is not worth it.

**Review by Caoxokhuyen — COMMENTED** [review]
5c1bd20f0d80bc71caf807432ba1c3936f3f3fb3

**Pull request merged**